docs/docs/getting_started/install_cli.md

```diff
@@ -5,7 +5,7 @@ sidebar_position: 10
 
 # Installing the Minder CLI
 
-The open source Minder CLI can communicate with either [Minder Cloud](https://cloud.stacklok.com/), Stacklok's free-to-use, cloud-hosted version of Minder, or with an on-premises version of Minder that has been [set up independently](../run_minder_server/run_the_server).
+The open source Minder CLI can communicate with either the free public instance provided by Stacklok api.stacklok.com, Stacklok's or with a self managed server [set up independently](../run_minder_server/run_the_server).
```

Comment:
```suggestion
The open source Minder CLI can communicate with either [the free public instance provided by Stacklok](../../#minder-public-instance), or with a [self managed server](../run_minder_server/run_the_server).
```

It feels like we should link to a central location for information about the public instance, rather than scattering mentions throughout the documentation.